pretyping/reductionops.ml

```diff
@@ -1110,6 +1110,34 @@ let sigma_univ_state =
     compare_instances = sigma_compare_instances;
     compare_cumul_instances = sigma_check_inductive_instances; }
 
+let univproblem_compare_sorts env pb s0 s1 uset =
+  let open UnivProblem in
+  match pb with
+  | Reduction.CONV -> UnivProblem.Set.add (ULe (Sorts.univ_of_sort s0, Sorts.univ_of_sort s1)) uset
+  | Reduction.CUMUL -> UnivProblem.Set.add (UEq (Sorts.univ_of_sort s0, Sorts.univ_of_sort s1)) uset
```

Comment:
```suggestion
  | Reduction.CONV -> UnivProblem.Set.add (UEq (Sorts.univ_of_sort s0, Sorts.univ_of_sort s1)) uset
  | Reduction.CUMUL -> UnivProblem.Set.add (ULe (Sorts.univ_of_sort s0, Sorts.univ_of_sort s1)) uset
```
x_x

Reply:
Copy-pasting is *hard*, you wouldn't believe!